### ISSUE TITLE
Improve the contributing guide at https://jenkins-x.io/contribute/development/

### ISSUE THREAD

**Issue by pmuir** [jenkins-x/jx#1817]
### Summary 

This guide is quite out of date (references Travis, refences Hugo?) and is really more of a guide for how to use Go and Git. If you are working on Jenkins X then https://github.com/jenkins-x/jx/blob/master/docs/contributing/hacking.md is of more use, but still missing stuff.

We should merge the good stuff from hacking into the main guide, and just 'symlink' them.

**Comment by pmuir** [comment]
I think so. The hacking.md file is pretty hidden right now. We may want to keep some minimal content in CONTRIBUTING.md as GitHub uses that as a 'special file' - see https://help.github.com/articles/setting-guidelines-for-repository-contributors/ - or it might make most sense to consolidate all of it in CONTRIBUTING.md and point the jx-docs page there?

**Comment by tgarlot** [comment]
Agreed. This is also what I prefer personnaly when I'm browsing a project on Github. 
So change would be:
- jx: "move" content of hacking.md to CONTRIBUTING.md (with small "refresh" and improvement). Remove hacking-md (or symlink it to CONTRIBUTION.md)
- jx-docs: create a CONTRIBUTION.md with the content of https://jenkins-x.io/contribute/documentation/ as in jx repository. Then link both contribution pages of jenkins-x.io to each CONTRIBTION.md on github.

hope I'm clear :-) 

Ok for that?

**Comment by tgarlot** [comment]
Finally, after a look at the content, I think it's better to keep a minimal CONTRIBUTING.MD that is pointing to the detailled guide on the website. Especialy as on the websire, we can have guides in many languaes (as in chinese today). 

Will start on this asap

**Comment by ccojocar** [comment]
@pmuir What is still required for this issue?

**Comment by pmuir** [comment]
@tgarlot I think this is done isn't it?
